src/openrct2-ui/windows/Ride.cpp

```diff
@@ -923,7 +923,7 @@ static void WindowRideDrawTabMain(rct_drawpixelinfo* dpi, rct_window* w)
  */
 static void WindowRideDrawTabVehicle(rct_drawpixelinfo* dpi, rct_window* w)
 {
-    WidgetIndex widgetIndex = WIDX_TAB_1 + WINDOW_RIDE_PAGE_VEHICLE;
+    WidgetIndex widgetIndex = WIDX_TAB_1 + static_cast<int32_t>(WINDOW_RIDE_PAGE_VEHICLE);
```

Comment:
Is it possible to redefine WINDOW_RIDE_PAGE_* and get rid of these casts, or would that require casts in other places?

Reply:
I think it would eventually require casts somewhere else, as it's being used to index into arrays and on these sum operations, etc.